elvis.config

```diff
@@ -11,7 +11,6 @@
                                                                         ]}}
                       , {elvis_style, line_length, #{limit => 80, skip_comments => false}}
                       , {elvis_style, operator_spaces, #{ rules => [ {right, ","}
-                                                                   , {right, "-"}
```

Comment:
Elvis was requesting a space for negative numbers, which I don't want. Not sure if there's a way to enforce space only when the `-` is used as a binary operator and not as a unary one.

Reply:
This is a bug, it shouldn't do that. 

There is a test case that checks for operators with one operand [here](https://github.com/inaka/elvis_core/pull/112/files#diff-1cb0d1d3d593a8deed4e80e20871c293R45). And the check is done [here](https://github.com/inaka/elvis_core/pull/112/files#diff-9d4ce82551a31e44e1d37fb5c5966363R292). 

Can you share the snippet of code that was getting the false positive?

Reply:
@jfacorro From `els_parser.erl`:

```
                      Range = #{ from => {StartLine, StartColumn}
                               , to   => {EndLine - 1, -1}
                               },
```